assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx

```diff
@@ -0,0 +1,120 @@
+// @flow
+
+// ----- Imports ----- //
+import React, { Component } from 'react';
+
+// ---- Types ----- //
+type StateTypes = {|
+  totalSoFar: number,
+  goal: number,
+|}
+
+type PropTypes = {}
+
+
+// ---- Helpers ----- //
+
+const getInitialTickerValues = (): Promise<{| totalSoFar: number, goal: number |}> =>
+  fetch('https://interactive.guim.co.uk/docsdata-test/1ySn7Ol2NQLvvSw_eAnVrPuuRnaGOxUmaUs6svtu_irU.json')
```

Comment:
do we need to handle the failure case?

Reply:
Would be better to have explicit handling but what happens if no data is returned (fetch fails or not in form expected) is the wrapper never gets `--hidden` removed and so a white space appears (see screenshot in PR description).